app/src/main/res/values/strings.xml

```diff
@@ -223,4 +224,19 @@
   <string name="add_profile_required">*Required</string>
   <string name="back_button">Back Button</string>
   <string name="next">Next</string>
+  <!-- AdministratorControlsFragment. -->
```

Comment:
Remove full stop from the end `<!-- AdministratorControlsFragment -->`

Reply:
Done